lib/msgpack/timestamp.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module MessagePack
+  Timestamp = Struct.new(:sec, :nsec)
+
+  class Timestamp # a.k.a. "TimeSpec"
+    TYPE = -1 # timestamp extension type
+
+    TIMESTAMP32_MAX_SEC = 0x100000000 # 32-bit unsigned int
```

Comment:
IMO "max" means the inclusive upper limit. For example, the max of `[0, 99]` is 99, but 98 for `[0, 99)`.
The max value of 32-bit bytes looks the 8 chars of `f`, not 9 chars.
How do you feel about it?

Reply:
Yep, it's confusing. `(1 << 32) - 1` is absolutely better.